dashboard/app/controllers/home_controller.rb

```diff
@@ -150,6 +150,7 @@ def init_homepage
       @homepage_data[:showCensusBanner] = show_census_banner
       @homepage_data[:showNpsSurvey] = show_nps_survey? SurveyResult::NET_PROMOTER_SCORE_2020
       @homepage_data[:donorBannerName] = donor_banner_name
+      @homepage_data[:schoolHasDonor] = !!current_user.school_donor_name
```

Comment:
Do you want to show this to all teachers whose schools are associated with a donor? Or only teachers in schools associated with Amazon?

Reply:
Also, having trouble wrapping my head around the distinction between donorBannerName and schoolHasDonor, can you help me understand? You're using a separate prop for this banner because you want a teacher to see the new banner even if they've dismissed the existing donor banner?

Reply:
Good call on that first question, the ticket said "Add a banner to the teacher homepage ONLY appears to teachers whose IDs are in our NCES spreadsheet" but let me check with Amanda. 

So I think `donorBannerName` is a bad variable name, it will return the donor name if there is a donor AND the teacher hasn't dismissed the donor banner in the past. Like you said, I still want this banner to show even if they've dismissed the other one.

Reply:
Good catch, it should just show when the donor is Amazon, I've pushed an update.